web/wp-content/themes/dctx-v2/docs/assets/js/vendor/fuse.min.js

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
:warning: **Warning**: File appears to be minified and cannot be processed (*Internal.Tokenizer.Exception*).